src/wiki/development/build-instructions.md

```diff
@@ -166,36 +166,56 @@ We recommend using a build workflow based on MSYS2, as it's the easiest way to g
 ### Dependencies
 
 - [MSYS2](https://www.msys2.org/) - Software Distribution and Building Platform for Windows
-  - Make sure to follow all instructions on the webpage.
 - [Java JDK 8 or later](https://adoptium.net/)
   - Make sure that "Set JAVA_HOME variable" is enabled in the Adoptium installer.
 
 ### Preparing MSYS2
 
-1. Open either the *MSYS2 MinGW CLANG x64* (reccomended), the *MSYS2 MinGW UCRT x64*, the *MSYS2 MinGW x64*, the *MSYS2 MinGW CLANG x86* or the *MSYS2 MinGW x86* shortcut from the start menu
+1. Open one of the shortcuts from the MSYS2 folder in the Start menu
 
-   - We recommend building using the CLANG msystems of MSYS2, as they compile considerably faster and with a few less bugs.
+   - We recommend building using the CLANG64 msystems of MSYS2, as they compile considerably faster and with a few less bugs.
 
 2. Install helpers: Run `pacman -Syu pactoys git mingw-w64-x86_64-binutils` in the MSYS2 shell.
-3. Install all build dependencies using `pacboy`: Run `pacboy -S toolchain:p cmake:p ninja:p qt6-base:p qt6-5compat:p qt6-svg:p qt6-imageformats:p quazip-qt6:p extra-cmake-modules:p ninja:p`.
+3. Install all build dependencies using `pacboy`: Run `pacboy -S toolchain:p cmake:p ninja:p qt6-base:p qt6-5compat:p qt6-svg:p qt6-imageformats:p quazip-qt6:p extra-cmake-modules:p ninja:p ccache:p`.
 
-   - Alternatively you can use Qt 5 (for older Windows versions), by running the following command instead: `pacboy -S toolchain:p cmake:p ninja:p qt5-base:p qt5-svg:p qt5-imageformats:p quazip-qt5:p extra-cmake-modules:p ninja:p`
+   - Alternatively you can use Qt 5 (for older Windows versions), by running the following command instead: `pacboy -S toolchain:p cmake:p ninja:p qt5-base:p qt5-svg:p qt5-imageformats:p quazip-qt5:p extra-cmake-modules:p ninja:p ccache:p`
    - This might take a while, as it will install Qt and all the build tools required.
 
 ### Compile from command line on Windows
 
-1. Clone Prism Launcher by doing `git clone --recursive https://github.com/PrismLauncher/PrismLauncher.git`, and change directory to the folder you cloned to.
-2. Now we can prepare the build itself: Run `cmake -Bbuild -DCMAKE_INSTALL_PREFIX=install -DENABLE_LTO=ON -DLauncher_QT_VERSION_MAJOR=6 -DCMAKE_OBJDUMP=/mingw64/bin/objdump.exe -G Ninja`. These options will copy the final build to `C:\msys64\home\<your username>\PrismLauncher\install` after the build.
+1. `cd` into the folder you cloned Prism Launcher to. Put quotes around the path.
```

Comment:
I removed the mention to clone the repo because it's already done at the top of the build instructions.
The repo doesn't actually have to be cloned within MSYS2. The user can clone it wherever.

Also, I say to put quotes around the path to prevent the `\`s from being read as escape sequences.
You could also replace them with `\\` or `/`, but just adding quotes around the path is easier.